packages/driver/cypress/fixtures/auth/idp.html

```diff
@@ -6,40 +6,62 @@
   <label for="name">Username: </label>
   <input data-cy="username" type="text" name="name" id="name" required>
   <button data-cy="login">Login</button>
+  <button data-cy="slow-login">Slow Login</button>
 
   <script>
-    document.querySelector('[data-cy="login"]').addEventListener('click', () => {
-
+    const createJwt = (username) => {
```

Comment:
a bit of a nit, but I wonder if we want to use the `token` vernacular over specifically calling the token explicitly a `jwt`, since this drifts a little bit from that signature.

Reply:
thats true, it's not really a jwt for sure since it's not signed